lib/elixir/lib/application.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Application do
   @moduledoc """
-  A module for working with applications and defining application callbacks.
+  Convenience functions for working with applications and defining
```

Comment:
I was trying to get rid of the repetititve "this modules does..." and  "a module for this and that".